src/webui/api/torrentscontroller.cpp

```diff
@@ -1129,17 +1129,23 @@ void TorrentsController::editTrackerAction()
     const auto id = BitTorrent::TorrentID::fromString(params()[u"hash"_s]);
     const QString origUrl = params()[u"origUrl"_s];
     const QString newUrl = params()[u"newUrl"_s];
+    // min tier is 0, so -1 indicates to keep tier unchanged
```

Comment:
Why not return "Bad request" in case of any invalid value is specified?

Reply:
I was referring to the inconsistent behavior of the current code. It returns an error for some invalid values (numbers < -1 and > 255), but silently ignores others (number -1 and any other non-numeric garbage).

Reply:
The code now correctly errors for non-integers and values less than 0.